src/components/Quiz/QuizzesStats.tsx

```diff
@@ -58,6 +68,38 @@ const QuizzesStats: React.FC = () => {
     JSON.parse(completed)
   )
 
+  // Data from Matomo, manually updated
+  const collectiveQuestionsAnswered = 100000
+  const collectiveAverageScore = 67.4 / 100 // converted to fraction for percentage format
+  const collectiveRetryRate = 15.6 / 100 // converted to fraction for percentage format
+
+  const formatNumber = new Intl.NumberFormat(localeForQuizStats, {
+    style: "decimal",
+    minimumSignificantDigits: 2,
+    maximumSignificantDigits: 3,
+  })
+
+  const formatPercent = new Intl.NumberFormat(localeForQuizStats, {
+    style: "percent",
+    minimumSignificantDigits: 2,
+    maximumSignificantDigits: 3,
+  })
```

Comment:
Wondering if it's worth pulling these out and importing them from the utils folder

Reply:
Agree! My bad, should have checked for that. 

We could refactor it to accept props for number of significant digits and use that across the codebase.